### DISCUSSION TITLE
When everyone I try to authorize my github account for a private repo a message pop up saying that I am mark as Spam

### DISCUSSION THREAD

**Discussion by FareIntelligence** [community/community#144648] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### GitHub Feature Area

Repositories

### Body

when I am trying to access a private repo from my github account (I am added as a colabarator) it shows a error of that my account is marked as spam and due to that I am unable to push or pull code from that repo, what is causing this issue????

**Comment by queenofcorgis** [comment, marked as answer]
Hi @FareIntelligence, 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

Please note: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
